dashboards/support.jsonnet

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env -S jsonnet -J ../vendor
```

Comment:
I had to ensure shebangs were added to executable files.

I removed the executable flag for the libsonnet file that renders blank anyhow.